contracts/migration/primitives/MigrationAccount.sol

```diff
@@ -3,6 +3,92 @@
 pragma solidity ^0.8.26;
 
 import {Account} from "contracts/extensions/account/Account.sol";
-import {EventEmitter} from "contracts/migration/EventEmitter.sol";
+import {AccountManagerPermissions, Transaction} from "contracts/extensions/account/IAccount.sol";
+import {SourceStamp, KeyValue} from "contracts/core/types/Types.sol";
+import {Errors} from "contracts/core/types/Errors.sol";
+import {ISource} from "contracts/core/interfaces/ISource.sol";
+import {WHITELISTED_MULTICALL_ADDRESS} from "contracts/migration/WhitelistedMulticall.sol";
```

Comment:
may be worth allowing this list to be added to just incase we have to add some more for some reason without needing to do another upgrade